### ISSUE TITLE
Flaky Integration Test: AccessLog TestAccessLogRotation

### ISSUE THREAD

**Issue by marco-jantke** [containous/traefik#2203]
### Do you want to request a *feature* or report a *bug*?

*bug*

### Summary

- *Which test is flaky?* -> `LogRotationSuite.TestAccessLogRotation`
- *Cause* -> Access Log file is not yet created after rotation.
- *Failed build* -> https://semaphoreci.com/containous/traefik/branches/master/builds/188

### Log excerpt

```
----------------------------------------------------------------------
FAIL: log_rotation_test.go:21: LogRotationSuite.TestAccessLogRotation

127.0.0.1 - - [26/Sep/2017:08:38:14 +0000] "GET /test1 HTTP/1.1" 200 22 - "Go-http-client/1.1" 1 "frontend1" "http://127.0.0.1:8081" 0ms
127.0.0.1 - - [26/Sep/2017:08:38:14 +0000] "GET /test1 HTTP/1.1" 200 22 - "Go-http-client/1.1" 2 "frontend1" "http://127.0.0.1:8081" 0ms
127.0.0.1 - - [26/Sep/2017:08:38:14 +0000] "GET /test1 HTTP/1.1" 200 22 - "Go-http-client/1.1" 3 "frontend1" "http://127.0.0.1:8081" 0ms
log_rotation_test.go:64:
    // Verify access.log output as expected
    lineCount = verifyLogLines(c, traefikTestAccessLogFile, lineCount, true)
log_rotation_test.go:128:
    c.Assert(err, checker.IsNil)
... value *os.PathError = &os.PathError{Op:"open", Path:"access.log", Err:0x2} ("open access.log: no such file or directory")
```

**Comment by marco-jantke** [comment]
Note: for this one I already have a fix. Will open PR next days.

**Comment by mmatur** [comment]
I close this issue because it is fixed by #2213
